### PR TITLE
Completing the Spanish translations + other fixes to dcat.ttl

### PR DESCRIPTION
**Pull request by agbeltran**
related to #174

### CODE REVIEW

**Comment by agbeltran** [comment]
Note that this PR addresses issue #961 in a ```skos:scopeNote``` in the turtle file.

**Comment by agbeltran** [comment]
Note that the ```skos:scopeNote``` about ```odrl:hasPolicy``` referred to ```See also guidance at § 10. License and rights statements.```, whose section is no longer $10. This has been fixed.

**Comment by agbeltran** [comment]
Note that the description of ```foaf:homepage``` was duplicated. Now fixed.

File: dcat/rdf/dcat.ttl [hunk 1]
```diff
@@ -844,14 +937,15 @@ dcat:mediaType
...
   skos:scopeNote "This property SHOULD be used when the media type of the distribution is defined in the IANA media types registry https://www.iana.org/assignments/media-types/, otherwise dct:format MAY be used with different values."@en ;
+  skos:scopeNote "Esta propiedad DEBERÍA usarse cuando el 'media type' de la distribución está definido en el registro IANA de 'media types' https://www.iana.org/assignments/media-types/, de lo contrario, dct:format PUEDE usarse con distintos valores."@es ;
   skos:scopeNote "Questa proprietà DEVE essere usata quando il tipo di media della distribuzione è definito nel registro dei tipi di media IANA https://www.iana.org/assignments/media-types/, altrimenti dct:format PUO 'essere usato con differenti valori"@it
 .
 dcat:compressFormat
   rdf:type rdf:Property ;
   rdf:type owl:ObjectProperty ;
   rdfs:comment "The format of the distribution in which the data is contained in a compressed form, e.g. to reduce the size of the downloadable file."@en ;
   rdfs:comment "El formato de la distribución en el que los datos están en forma comprimida, e.g. para reducir el tamaño del archivo a bajar."@es ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]: if the PR #986 is merged, the spanish translation should change as in the following
```suggestion
  rdfs:comment "El formato del archivo en el que los datos están en forma comprimida, e.g. para reducir el tamaño del archivo a bajar."@es ;
```

**Review by riccardoAlbertoni — APPROVED** [review]

File: dcat/rdf/dcat.ttl (hunk 1 above)
Comment (agbeltran) [reply dcat/rdf/dcat.ttl]: Thanks @riccardoAlbertoni - I'll check #986

**Review by davebrowning — CHANGES_REQUESTED** [review]
@agbeltran - this looks fine except it has clashed with PR #988  (and needs co-ordination with #987) so needs a little tidy up....

**Comment by agbeltran** [comment]
thanks @davebrowning - this PR was ready before those other ones; indeed now there are some conflicts which I will look at

**Comment by agbeltran** [comment]
@davebrowning this is ready for another review.- thanks!

**Review by davebrowning — APPROVED** [review]

**Comment by davebrowning** [comment]
Thanks @agbeltran - you're right, we (actually I) got a bit out of sync.  Thanks for fixing.

**Pull request merged**